paywall/src/static/adremover/integrationtesting-loggedin.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Lorem Ipsum Daily</title>
+    <script src="./hookupGanache.js"></script>
+    <script>
+      // we use 4 parameters from query:
```

Comment:
Although this is for testing, we should extract to code living inside the script tags on this page into its own library so that we  can test, modify, review, etc these if it comes down to  it